pkg/daemons/control/deps/deps.go

```diff
@@ -440,6 +446,25 @@ func genServerCerts(config *config.Control) error {
 		return err
 	}
 
+	altNames = &certutil.AltNames{
+		DNSNames: []string{"localhost"},
+	}
+	addSANs(altNames, []string{"127.0.0.1", "::1"})
```

Comment:
This is super nitty but you could just keep all the names in one go:
```suggestion
	altNames = &certutil.AltNames{}
	addSANs(altNames, []string{"localhost" ,"127.0.0.1", "::1"})
```